stable/mongodb/templates/_helpers.tpl

```diff
@@ -19,7 +19,8 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 Return the proper image name
 */}}
 {{- define "mongodb.image" -}}
-{{- $registryName :=  default .Values.image.registry "docker.io" -}}
-{{- $tag := .Values.image.tag | default "latest" -}}
-{{- printf "%s/%s:%s" $registryName .Values.image.repository $tag -}}
+{{- $registryName :=  .Values.image.registry -}}
+{{- $repositoryName := .Values.image.repository -}}
+{{- $tag := .Values.image.tag -}}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
```

Comment:
if `registry = ""` this would be `/bitnami/mongodb:3.7.3`, is that okay, or should we work in a special case?

Reply:
Looking at `values.yaml`, I'm not sure it is possible for `registry` to be an empty string without it being deliberately set as so:

```yaml
image:
  registry: docker.io
  repository: bitnami/mongodb
  tag: 3.7.3
```

Of course, if I'm missing something here and it is possible for `image.registry` to be an empty string then I agree, there should be a special case (perhaps a use of `default`) and I'm happy to make this change, but I did run some test installs and the computed values looked OK:

```yaml
# running `helm install --debug --dry-run stable/mongodb`
COMPUTED VALUES:
image:
  pullPolicy: IfNotPresent
  registry: docker.io
  repository: bitnami/mongodb
  tag: 3.7.3
```

```yaml
# running `helm install stable/mongodb` and inspecting the deployment resource (`kubectl get deployment "${release_name}" -o yaml`)
...
    spec:
      containers:
        - ...
          image: /bitnami/mongodb:3.7.3
...
```

Reply:
@twistedvines I guess I was thinking about a case where someone would explicitly set `registry = ""`, but now that I think about it if you're doing that the default `docker.io` should work as well.